machine/wnc_x86_64/wnc.patch/installer/x86_64/install-arch~

```diff
@@ -0,0 +1,499 @@
+# x86_64 specific ONIE installer functions
```

Comment:
This is a back up file from your editor.  Please do not commit edit back up files.